docs/concepts/staking/block_reward.md

```diff
@@ -1,21 +1,70 @@
 ---
 tags:
+  - block reward
   - Staking
 ---
 
 
 # Staking
 
 
-## Block reward
+## Block Reward
+---
+
+Block rewards in a blockchain serve as an incentive for validators who actively contribute to the creation of new blocks within the blockchain network. This rewarding process, commonly referred to as validation, is particularly relevant in **delegated proof-of-stake (DPoS)** blockchains.
+
+In DPoS blockchains, validators are selected to validate transactions and generate new blocks based on the quantity of XPLL they have staked within the network. As a reward for their contributions, validators receive a share of the transaction fees and a portion of the block rewards for successfully creating new blocks. 
+
+Block rewards play a crucial role in blockchain networks by serving multiple purposes. They incentivize miners and validators' participation in the network's consensus process, thereby upholding the security and integrity of the network. Additionally, block rewards facilitate the introduction of new XPLL into circulation, ensuring the long-term sustainability of the network.
```

Comment:
The term "miners" isn't quite applicable to ParallelChain's context since operate DPoS, so there isn't mining, strictly speaking. Maybe something along the lines of the "proposal and voting on blocks by validators"? 
Taking one step back, this part seems to be repeating the point on incentives on line 14, maybe better to merge them into a single paragraph?